cooltools/dotfinder.py

```diff
@@ -1341,7 +1342,9 @@ def scoring_step(
             # "create_cooler" API needs more consistency:
             # https://github.com/mirnylab/cooler/issues/149
             #  manually describing "data"-columns (except for bin_ids).
-            columns = ["count",] + ["la_exp." + k + ".value" for k in kernels]
+            columns = [
+                "count",
```

Comment:
sometime `black` does strange things - like this one ...
I hope, at least , that it stays consistent from run to run, from version to version